frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -99,16 +103,9 @@ export const detectOCS: FeatureDetector = async (dispatch) => {
 
 export const detectOCSSupportedFeatures: FeatureDetector = async (dispatch) => {
   try {
-    const subscription = await fetchK8s<K8sResourceKind>(
-      SubscriptionModel,
-      'ocs-subscription',
-      CEPH_STORAGE_NAMESPACE,
-    );
-    const ocsCSV = await fetchK8s(
-      ClusterServiceVersionModel,
-      subscription?.status?.currentCSV,
-      CEPH_STORAGE_NAMESPACE,
-    );
+    const csvList: any = await fetchK8s(ClusterServiceVersionModel, '', CEPH_STORAGE_NAMESPACE);
+    const csvItems = csvList.items as ClusterServiceVersionKind[];
+    const ocsCSV = csvItems.find((obj) => getName(obj).includes(OCS_OPERATOR));
```

Comment:
Substring match seems imprecise. Do we not know the full name?

Reply:
@spadgett OCS CSV name changes every release, in the sense the version of OCS is added to the name, like `ocs-operator.v4.6.0` for 4.6 etc, so we cannot check for full name here. Probably instead of substring, we can check with `startsWith` function, wdyt?

Reply:
I think that would be better 👍